core-bundle/contao/models/PageModel.php

```diff
@@ -997,6 +997,10 @@ public function loadDetails()
 			$this->mailerTransport = $objParentPage->mailerTransport;
 			$this->enableCanonical = $objParentPage->enableCanonical;
 			$this->maintenanceMode = $objParentPage->maintenanceMode;
+			$this->enableCsp = $objParentPage->enableCsp;
+			$this->csp = $objParentPage->csp;
+			$this->cspReportOnly = $objParentPage->cspReportOnly;
+			$this->cspReportLog = $objParentPage->cspReportLog;
```

Comment:
It seems that the properties are not annotated in the `PageModel` class?

<img width="599" alt="" src="https://github.com/contao/contao/assets/1192057/b19de68e-6266-4ac2-ae8f-25c64a425f69">

Reply:
Fixed in a9f7b3b842